app/src/main/java/eu/kanade/tachiyomi/extension/ExtensionManager.kt

```diff
@@ -340,9 +340,12 @@ class ExtensionManager(
         override fun onExtensionUntrusted(extension: Extension.Untrusted) {
             val installedExtension = _installedExtensionsFlow.value
                 .find { it.pkgName == extension.pkgName }
-                ?: return
-            _installedExtensionsFlow.value -= installedExtension
-            _untrustedExtensionsFlow.value += extension
+
+            if (installedExtension != null) {
+                _installedExtensionsFlow.value -= installedExtension
+            } else {
+                _untrustedExtensionsFlow.value += extension
+            }
```

Comment:
The extension won't appear as untrusted after updating 🤔

Reply:
it does show up